src/adapters/postproc.rs

```diff
@@ -145,38 +143,57 @@ pub fn postproc_prefix(line_prefix: &str, inp: impl AsyncRead + Send) -> impl As
     StreamReader::new(oup_stream)
 }
 
-/**
- * adds the prefix `Page N:` to each line,
- * where N starts at one and is incremented for each ASCII Form Feed character in the input stream.
- * (That's the format output by pdftotext)
- */
-pub fn postproc_pagebreaks(line_prefix: &str, inp: impl AsyncRead) -> impl AsyncRead {
-    let line_prefix = line_prefix.to_string(); // clone since
-    let mut page_count = 1;
+/// Adds the prefix "Page N:" to each line,
+/// where N starts at one and is incremented for each ASCII Form Feed character in the input stream.
+/// ASCII form feeds are the page delimiters output by `pdftotext`.
+pub fn postproc_pagebreaks(input: impl AsyncRead + Send) -> impl AsyncRead + Send {
```

Comment:
I couldn't get this working with the `line_prefix: &str` parameter. Compiler was complaining about an implicit lifetime on that parameter that I couldn't figure out.

Reply:
Added in https://github.com/lafrenierejm/ripgrep-all/commit/4cac831a8f6a8f308ef940fdcc79cc92175009a5.